spec/javascripts/checkbox-summary_Spec.js

```diff
@@ -0,0 +1,95 @@
+describe('CheckboxSummary', function() {
+  var $fixture, subject;
+
+  beforeEach(function() {
+    $fixture = $('<div class="CheckboxSummary"><input type="checkbox" value="Orange" id="Orange" /><input type="checkbox" value="Green" id="Green" /><span class="CheckboxSummary-summary"></span></div>'),
+    subject = new moj.Modules._CheckboxSummary($fixture);
+  });
+
+  it('is an instantiable object', function() {
+    expect(typeof subject).toBe('object');
+  });
+
+  it('is rendered by moj.Events module', function() {
+    $fixture = $('<div class="CheckboxSummary"><input type="checkbox" value="Orange" id="Orange" /><input type="checkbox" value="Green" id="Green" checked="checked" /><span class="CheckboxSummary-summary"></span></div>'),
+    subject = new moj.Modules._CheckboxSummary($fixture);
+    moj.Events.trigger('render');
+    expect($fixture.find('.CheckboxSummary-summary').text()).toBe('Green');
+  });
+
+  it('caches elements for reuse', function() {
+    expect(subject.$checkboxes.length).toBe(2);
+    expect(subject.$summaries.length).toBe(1);
+  });
+
+  it('creates a summary from currently checked boxes', function() {
+    $fixture.find('#Orange').prop('checked', true).change();
+    expect($fixture.find('.CheckboxSummary-summary').text()).toBe('Orange');
+    expect(subject.getChecked(subject.$checkboxes).length).toBe(1);
+  });
+
+  it('removes text from summaries when checked selections are un-checked', function() {
+    $fixture.find('#Orange, #Green').prop('checked', true).change();
+    expect($fixture.find('.CheckboxSummary-summary').text()).toBe('Orange, Green');
+    $fixture.find('#Orange').prop('checked', false).change();
+    expect($fixture.find('.CheckboxSummary-summary').text()).toBe('Green');
+  });
+
+  it('uses checkbox values as summary text', function() {
+    $fixture.find('#Orange').prop('checked', true).change();
+    expect($fixture.find('.CheckboxSummary-summary').text()).toBe('Orange');
+  });
```

Comment:
Isn't this test a subset of `it('creates a summary from currently checked boxes'`? Can it be removed and just keep that one?